pkg/project/project.go

```diff
@@ -53,13 +53,14 @@ func Create(client *occlient.Client, projectName string) error {
 	return nil
 }
 
-// Delete deletes the project with name projectName and sets the project with name currentProject as current project and returns errors if any
-func Delete(client *occlient.Client, projectName string) error {
+// Delete deletes the project with name projectName and sets the project with name currentProject as current project
+// and returns the current project or ("" if no current project is set) and errors if any
+func Delete(client *occlient.Client, projectName string) (string, error) {
```

Comment:
This `Delete` function is weird but it's off-topic for this PR. I just find the logic very convoluted. Will look into it at some point.

Reply:
ok :+1: